lib/vericite.rb

```diff
@@ -273,7 +273,7 @@ def sendRequest(command, args)
           end
           data.each do |externalContentUploadInfo|
             #API will return an upload URL to store the submission (throws an exception if it fails)
-            res = api_client.uploadfile(externalContentUploadInfo.url_post, args[:pdata])
+            res = api_client.uploadfile(externalContentUploadInfo.url_post, args[:pdata], externalContentUploadInfo.headers)
```

Comment:
Our linter noticed that res is an unused variable, and I concur.  Should this be assigned to the response from above?  If its truly not used (and i realize the existing code was like that) we should probably just ignore the return value.